### DISCUSSION TITLE
Erro de BUILD no curso "Acesso à dados com .NET, C#, Dapper e SQL Server"

### DISCUSSION THREAD

**Discussion by ghost** [balta-io/forum#347] [Tira Dúvidas] [answered]
O erro ocorre quando tento rodar o progama, mesmo fazendo ref (using Blog.Repositories) a classe Program acaba não passando/reconhecendo a outra classe.
![Captura de Tela (58)](https://github.com/balta-io/forum/assets/99696429/2386e9b4-a406-44e9-bca6-474ab7b94942)

**Comment by andrebaltieri** [comment, marked as answer]
@leticialbuk por favor, tenta fazer um `dotnet clean` e `dotnet build` novamente, as vezes fica sujeira no cache do Windows.
